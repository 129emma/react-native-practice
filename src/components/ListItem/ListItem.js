import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const listItem = (props) => (
  <View style = {styles.listItem} onPress={props.onItemPressed}>
    <Text>
    {props.placeName}
    </Text>
  </View>
);

const styles = StyleSheet.create({
  listItem: {
    borderLeftColor: 'transparent',
    borderRightColor: 'transparent',
    borderTopColor: 'transparent',
    borderBottomColor: 'red',
    borderWidth: 0.5

  }
});

export default listItem;
