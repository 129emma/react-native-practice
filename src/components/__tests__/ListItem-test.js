import React from 'react';
import renderder from 'react-test-renderer';

import ListItem from '../ListItem/ListItem';

it('renders ListItem using Snapshots', () => {
    const listItem = renderder.create(<ListItem />).toJSON();
        expect(listItem).toMatchSnapshot();
});

