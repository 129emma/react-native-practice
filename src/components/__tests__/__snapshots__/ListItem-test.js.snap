// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ListItem using Snapshots 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "lightblue",
        "borderColor": "lightblue",
        "borderRadius": 4,
        "flexDirection": "row",
        "marginBottom": 10,
        "padding": 10,
      }
    }
  >
    <Image
      source={undefined}
      style={
        Object {
          "height": 30,
          "marginRight": 20,
          "width": 30,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "textAlign": "center",
        }
      }
    />
  </View>
</View>
`;
